src/parser/parser.yy

```diff
@@ -3139,6 +3139,22 @@ admin_job_sentence
          delete hl;
          $$ = sentence;
     }
+    | KW_SUBMIT KW_JOB KW_BALANCE KW_ZONE {
```

Comment:
Since there are two mode, maybe modify the command as follows:
submit job balance in zone [remove host_list]
submit job balance across zone [remove zone_list]

Reply:
> Since there are two mode, maybe modify the command as follows: submit job balance in zone [remove host_list] submit job balance across zone [remove zone_list]

done